kotlin/android/src/main/java/com/reactnativesecurekeystore/SecureKeystoreImpl.kt

```
@@ -306,12 +281,62 @@ class SecureKeystoreImpl(
+                    keyPair.add(publicKey)
+                } else {
+                    Log.e("SecureKeystore", "Biometric authentication failed")
+                }
+            } else {
+                val privateKey = preferences.getPreference(privateKeyAlias, "")
+                val publicKey = preferences.getPreference(publicKeyAlias, "")
+                keyPair.add(privateKey)
+                keyPair.add(publicKey)
+            }
+            return keyPair
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)